data/metric/metrics.go

```diff
@@ -50,6 +50,35 @@ type summary struct {
 	Max     *float64 `json:"max"`
 }
 
+type bucket struct {
```

Comment:
Would make sense to use directly [the prometheus library](https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go) instead of our implementation?
 Or we are doing so not to be locked with such implementation/standard? In that case I would not call the struct `PrometheusHistogram` but simply `Histogram`

Reply:
We could, but I wanted to avoid the dependency in here and in the agent. And this way we control the structure better.
We use the prefix Prometheus first because there is already a Summary metric from NR, and second because the agent needs to transform it and so if we use a "generic" name it could later conflict with other histogram implementation

Reply:
Fine for me! 👍